config.cmake

```diff
@@ -307,12 +312,6 @@ ENDMACRO()
 MACRO(ADD_MODULE_LIBRARY type libname version_string major)
     set(dflags "")
```

Comment:
It turns out that this variable is not needed. And those places where it is used.
It is also necessary to delete it all.

Reply:
i've removed them, but now, DLL is not installed anymore

Reply:
because, i think, there is no more "install TARGETS" anymore